api/types/types.go

```diff
@@ -161,6 +161,7 @@ type PacketInjection struct {
 	IncrementPayload int64 `yaml:"IncrementPayload"`
 	StartTime        time.Time
 	Pcap             []byte `yaml:"Pcap"`
+	TTL              int64  `yaml:"TTL"`
```

Comment:
TTL should be a `unint8` (0-255)

https://en.wikipedia.org/wiki/Time_to_live

```
The maximum TTL value is 255, the maximum value of a single octet
```

Reply:
I agree this will avoid uint8(ttl) later

Reply:
done!